src/adapters/bidderFactory.js

```diff
@@ -250,6 +256,17 @@ export function newBidder(spec) {
     }
   });
 
+  function isUnknownBidder(responseBidder, requestBidder, adapterCode) {
+    let currentBidderSettings = !!bidderSettings.get(requestBidder, 'allowUnknownBidderCodes', adapterCode || null);
```

Comment:
Sorry for not mentioning this earlier - I don't think `currentBidderSettings` is a good name; `allowUknownBidders` or similar would be a better one.

Reply:
Done :)